.github/workflows/WorkitemValidation.yaml

```diff
@@ -15,16 +15,46 @@ defaults:
 
 jobs:
   WorkItemValidation:
+    name: 'Validate link to issues'
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
         uses: actions/checkout@v3
 
+      - name: Link work items to pull request if possible
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          build/scripts/PullRequestValidation/LinkPullRequestToWorkItem.ps1 -PullRequestNumber ${{ github.event.pull_request.number }} -Repository ${{ github.repository }}
+
       - name: Validate work items for pull request
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
           build/scripts/PullRequestValidation/ValidateIssuesForPullRequest.ps1 -PullRequestNumber ${{ github.event.pull_request.number }} -Repository ${{ github.repository }}
+  
+  WorkitemValidationForMicrosoft:
+    name: 'For Microsoft: Validate link to internal workitems' 
+    runs-on: ubuntu-latest
+    needs: WorkItemValidation
```

Comment:
Why not get the two jobs run in parallel? These are two different and independent checks.

Reply:
Mmh.. I see your point. However, WorkItemValidationForMicrosoft will try to find AB# references in the linked issue. Wouldn't it be nice to know that the issue is validated (i.e. open and approved) before we do this?

Reply:
Yeah, that's good point, let's keep the chain.